source/hands_on/wqmix.rst

```diff
@@ -0,0 +1,155 @@
+WQMIX
+^^^^^^^
+
+Overview
+---------
+WQMIX was first proposed in `Weighted QMIX: Expanding Monotonic Value Function Factorisation for Deep Multi-Agent Reinforcement Learning <https://arxiv.org/abs/2006.10800>`_.
+Their focus of study is the unique properties of the particular function space represented by monotonic structure of Q functions in Qmix,
+and the tradeoffs in representation quality across different joint actions induced by projection into that space.
+They show that the projection in QMIX can fail to recover the optimal policy,
+which primarily stems from the equal weighting placed on each joint action.
```

Comment:
use unordered list can be more clear

Reply:
Use the same format of qmix on lines 7 and 9